routes/index.js

```diff
@@ -12,4 +13,12 @@ router.get('/', function(request, response, next) {
   response.render('index', { title: 'Express', attempts: numberGuesser.guesses });
 });
 
+router.get('/scrabBag', function(request, response, next) {
+  const { playedLetters, exit } = request.query
+  scrabBag.clear()
```

Comment:
Here, we should be creating an instance of the `ScrabbleBag` class, and using that instance:
```js
const bag = new ScrabbleBag();
// No need to clear, this is a fresh instance
bag.play( playedLetters )
// Use some function to construct the partitioned tile amounts, probably should exist in the scrabble bag module:
const tiles = tilePartition( bag.tiles() )
// render
```